IFC4x3/PropertyConstants/s/SINGLE_0GOpSfSzvF7wmhqiCW3KVg.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SINGLE_0GOpSfSzvF7wmhqiCW3KVg" Name="SINGLE" UniqueId="10633729-73de-4f1f-ac2b-d2c3200d47ea" />
```

Comment:
Documentation?

Reply:
Not used anymore in railway. Now it's SINGLE_STAY.
Can't erased this constant because it is used in other enum.